Jenkinsfile

```diff
@@ -3,19 +3,14 @@ pipeline {
     stages {
         stage('build') {
             steps {
-                 sh '''
-                    npm install -g typescript
-                    npm install
-                    npm run vscode:prepublish
-                    '''
+                sh 'npm install'
```

Comment:
the Jenkinsfile isn't used, not sure if it's worth keeping?

Reply:
Let me remove it in a separate PR then :)